src/main/java/com/example/task_2/Ticket.java

```diff
@@ -39,29 +45,75 @@ public Ticket(String id, String concertHall, short eventCode, boolean isPromo, c
         this.price = price;
 
         setCreationTime();
+        processNullableWarningAnnotation();
     }
 
-    private void setCreationTime() {
-        this.unixTime = LocalDateTime.now().atZone(ZoneId.systemDefault()).toInstant().toEpochMilli();
-    };
+    public void setUnixTime(long unixTime) { //possibility to change time
+        this.unixTime = unixTime;
+    }
+
+    public void setStadiumSector(char stadiumSector) { // method to change stadium sector
```

Comment:
There is no big sense in comments like this we understand what this method doing from his name. Use comments to describe some arduous logic instead